manifests/bases/hr/deployment.yaml

```diff
@@ -23,8 +23,8 @@ spec:
         - name: github-container-registry
```

Comment:
(philosophical point that probably doesn't belong in a PR review):

I'm a little bit sad that we had to go down the base+overlays route, because of all the [incidental complexity](https://blog.darklang.com/what-is-dark/#accidental-complexity) that it involves. After the hard-learned lessons of going over-the-top with infra complexity on the FutureNHS project, I am very wary of complexity creep in kubernetes setups.

The nirvana that I was hoping for with dapr is that everyone in the team knows the runes required to spin up all services locally **without** docker (in a hot-reloading fashion if desired) and then we have a single production cluster that's configured with flat kubernetes manifests.

I suppose the thing that overlays give is is the ability to test the whole shebang out on a completely separate non-production environment so we're confident that our manifest changes are good before shipping them to prod, so this PR is a net-win, but it is a trade-off.

It's probably worth keeping an eye out to see if there is a third way, that can give us the confidence without the complexity.

Reply:
Yeah I hate complexity, I'm not clever enough to work with it!

I think its true that this point will get lost in a pr review, so I have created an issue
https://github.com/redbadger/badger-brian/issues/14